content/english/blog/globally_distributed_webassembly_applications_with_wasmcloud_and_nats.md

```diff
@@ -0,0 +1,364 @@
+---
+title : "Globally Distributed WebAssembly Applications with wasmCloud and NATS"
+image : "images/blogs/ngs-global.png"
+date: 2022-06-30T9:00:00-04:00
+author: "Brooks Townsend"
+author_profile: "https://linkedin.com/in/brooks-townsend"
+description : "Taking a wasmCloud lattice from local to globally distributed with NATS and NGS"
+categories: ["webassembly", "wasmcloud", "nats", "distributed", "lattice"]
+draft : false
+---
+
+The first claim we make about wasmCloud on our documentation site is: "wasmCloud is a distributed platform..." The best definition I could find, on [Wikipedia](https://en.wikipedia.org/wiki/Distributed_computing) of course, is:
+
+{{< aside >}}
+Distributed computing is a field of computer science that studies distributed systems. A distributed system is a system whose components are located on different networked computers, which communicate and coordinate their actions by passing messages to one another from any system.
+{{< /aside >}}
+
+So, by the definition, as soon as we got two WebAssembly modules talking to each other on different networked computers we had a distributed system. Of course, we didn't stop there, and today we're going to walk through how you can run a global wasmCloud [lattice](https://wasmcloud.dev/reference/lattice/) using NATS and NGS.
+
+# How we got to NATS
+
+Back in the 0.18.0 days of wasmCloud we supported local host process calls which allowed developers to avoid installing NATS. Now, all wasmCloud hosts run atop NATS as a networking infrastructure that we call a [lattice](https://wasmcloud.dev/reference/lattice/). We've taken a stance *"compatible with, but not dependent upon"* for as much as possible (Kubernetes, Docker, bare metal, IoT, nomad) so this is a significant choice, and today you'll see the reasons why.
+
+  
+
+We're going to do a brief introduction on the power of NATS, talk about how wasmCloud uses it, and then get into configuring it to connect wasmCloud compute anywhere.
+
+  
+
+# NATS, Leaf Nodes, and NGS (oh my!)
```

Comment:
nit: the 3-item list + `oh my!` thing may be a bit overused. Fine to keep it, but I'd be okay with losing the `(oh my!)` as well.